site/public/templates/grading/EditGradeable.twig

```diff
@@ -7,11 +7,14 @@
 
 {% block components_block %}
     {% for component in gradeable.components %}
-        <div class="component-container edit-component-container{{ (component.peer_component) ? ' peer-component-container' : '' }}" {{ (component.peer_component) ? 'data-testid="peer-component-container"' : '' }}>
```

Comment:
The testid is lost in this conversion, if you think that this is a better way of displaying it, make sure no information is lot.